requirements.dev.txt

```diff
@@ -21,3 +21,4 @@ selenium==4.7.2
 sqlalchemy-stubs>=0.3
 tenacity==7.0.0
 xlrd==2.0.1
+sentry-sdk[flask]
```

Comment:
```suggestion
```
you only need this in `requirements.api.txt` -- the only time `requirements.dev.txt` is used, its combined with `requirements.api.txt`: https://github.com/cmu-delphi/delphi-epidata/blob/08de322948ac6d55ad394d7c2fab62cbd257372b/dev/docker/python/Dockerfile#L13